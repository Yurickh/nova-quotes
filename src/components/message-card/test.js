import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import MessageCard from '.'

describe('MessageCard', () => {
  it('should render correctly', () => {
    const tree = shallow(<MessageCard message="Test message"/>)
    expect(toJSON(tree)).toMatchSnapshot()
  })
})
