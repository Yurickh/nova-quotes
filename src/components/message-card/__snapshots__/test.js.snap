// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageCard should render correctly 1`] = `
<div
  className="message-card"
>
  <div
    className="username"
  >
    Autor desconhecido
  </div>
  <p
    className="message"
  >
    Test message
  </p>
</div>
`;
